homeassistant/components/prometheus/__init__.py

```diff
@@ -647,6 +647,15 @@ def _handle_counter(self, state):
 
         metric.labels(**self._labels(state)).set(self.state_as_number(state))
 
+    def _handle_update(self, state):
+        metric = self._metric(
+            "update_state",
+            self.prometheus_cli.Gauge,
+            "State of the update sensor (0/1)",
```

Comment:
I am not familiar with the `update` entities, but I've not noticed the term "update sensor" in [the docs](https://www.home-assistant.io/integrations/update/). I'd suggest changing the metric description to something that will use terminology consistent with the docs, and will help users understand the meaning of the metric. For example, "Update state, indicating if an update is available (0/1)"

Reply:
I think you're right—I've not chosen the right words here. I'll fix this PR up.